pyccel/codegen/printing/cucode.py

```diff
@@ -61,14 +61,45 @@ def _print_Module(self, expr):
             else:
                 local_imports += self._print(imp)
 
-        imports = f'{c_headers_imports}\
-                    extern "C"{{\n\
-                    {local_imports}\
-                    }}'
+        imports = (f'{c_headers_imports}\n'
+                   f'{local_imports}\n')
 
         code = f'{imports}\n\
                  {global_variables}\n\
                  {body}\n'
 
         self.exit_scope()
         return code
+
+    def _print_ModuleHeader(self, expr):
+        self.set_scope(expr.module.scope)
+        self._in_header = True
+        name = expr.module.name
+        if isinstance(name, AsName):
+            name = name.name
```

Comment:
Are there cases where the name is an `AsName` and cases where is not an `AsName`?

Reply:
After checking the code, I don't think that check is necessary for now. I will remove it until I need it

Reply:
[b4b3cce](https://github.com/pyccel/pyccel-cuda/pull/49/commits/b4b3cceb6ce0fbaed4ff906ecfb1f036f8bc4af8)